crcCards/StudyBlock.txt

```diff
@@ -2,7 +2,8 @@ Class name: StudyBlock
 
 Responsibilities:
 * Creates a study block
+* Blocks off time in schedule/ calendar
 
 Collaborators:
 * TaskManager
-* Scheduler
\ No newline at end of file
+* Scheduler
```

Comment:
Also interacts with StudyMethods (see Jeb's scenario on the slides) in order to figure out how to organize the study block